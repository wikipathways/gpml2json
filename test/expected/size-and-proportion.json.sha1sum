b63131183fb23a702d01b61ff8a041e862ce6467 ?-
