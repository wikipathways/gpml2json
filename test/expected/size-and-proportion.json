{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["id3b9f67a","cbe54","bd26b","fbba0","ea535","bb9f7","c3d22","d0ad7","d3372","c87b5","c70e2","eca55","id41348e99","fa68f","ca2cc","ea33b","debe1","ba4c6","ad9d9","ec998","ae3aa","eafdf","c3aac","fae6b","ffa32","a5963","f2f16","d8736","c6ccb","bb5ba","f5713","b505f","c43a3","be35d","db8d7","b68ac","e7b76","ff5a5","b32c8"],"drawAs":"rect","gpmlElementName":"Pathway","height":4804.444482440042,"kaavioType":"Group","name":"test case","padding":5,"type":["Pathway"],"width":1678.6187845303864,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","dataSourceVersion":"20130813","standardName":"test case","displayName":"test case","textContent":"Name: test case","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=test%20case&doSearch=1","id":"hash://sha256/8f6bee1805dc0aacfdb853d5c49256305e4a4008c267084ad35519d9dbc4dec9"},"entityMap":{"debe1":{"id":"debe1","x":1085.6479065433005,"y":1038.095238095238,"fontSize":10,"height":521.4192495028547,"rotation":0,"drawAs":"rect","verticalAlign":"middle","width":562.9708779870859,"zIndex":16384,"textAlign":"center","color":"#000000","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"bb5ba":{"id":"bb5ba","textContent":"PathVisio-Java has some bugs with displaying CenterX and\nCenterY values, as seen in this file.\n\nThe dashed lines represent the bounding boxes as per the\nvalues set in the PathVisio-Java Properties panel and as\nwritten to GPML.\n\nThe solid lines represent the bounding boxes as actually\ndisplayed in PathVisio-Java.","textAlign":"left","x":620.5080887080557,"y":22.59887005649719,"backgroundColor":"transparent","fontSize":16,"height":170.90395480225988,"width":444.46366270437375,"zIndex":33000,"color":"#000000","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":0,"drawAs":"none","verticalAlign":"top","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Label","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Label"},"ba4c6":{"id":"ba4c6","x":100,"y":300,"color":"#ff0000","fontSize":10,"height":600,"rotation":0,"drawAs":"rect","verticalAlign":"middle","width":600,"zIndex":32969,"textAlign":"center","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"ad9d9":{"id":"ad9d9","x":100,"y":3100,"color":"#ff0000","fontSize":10,"height":600,"rotation":0,"drawAs":"Ellipse","verticalAlign":"middle","width":600,"zIndex":32979,"textAlign":"center","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"ec998":{"id":"ec998","x":100,"y":1000,"color":"#ff0000","fontSize":10,"height":600,"rotation":0,"drawAs":"ArcPathVisio","verticalAlign":"middle","width":600,"zIndex":32981,"textAlign":"center","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"ae3aa":{"id":"ae3aa","x":900,"y":1000,"color":"#ff0000","fontSize":10,"height":600,"rotation":0,"drawAs":"TrianglePathVisio","verticalAlign":"middle","width":600,"zIndex":32983,"textAlign":"center","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"eafdf":{"id":"eafdf","x":100,"y":3800,"color":"#ff0000","fontSize":10,"height":600,"rotation":0,"drawAs":"PentagonPathVisio","verticalAlign":"middle","width":600,"zIndex":32985,"textAlign":"center","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"c3aac":{"id":"c3aac","x":100,"y":1700,"color":"#ff0000","fontSize":10,"height":600,"rotation":0,"drawAs":"HexagonPathVisio","verticalAlign":"middle","width":600,"zIndex":32987,"textAlign":"center","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"fae6b":{"id":"fae6b","x":900,"y":1700,"color":"#ff0000","fontSize":10,"height":600,"rotation":0,"drawAs":"Brace","verticalAlign":"middle","width":600,"zIndex":32989,"textAlign":"center","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"ffa32":{"id":"ffa32","cellularComponent":"Mitochondria","x":900,"y":3100,"color":"#ff0000","fontSize":10,"height":600,"rotation":0,"drawAs":"Mitochondria","verticalAlign":"middle","width":600,"zIndex":32991,"textAlign":"center","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode","PhysicalEntity","CellularComponent","Mitochondria"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"a5963":{"id":"a5963","cellularComponent":"SarcoplasmicReticulum","x":100,"y":2400,"color":"#ff0000","fontSize":10,"height":600,"rotation":0,"drawAs":"SarcoplasmicReticulum","verticalAlign":"middle","width":600,"zIndex":32993,"textAlign":"center","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode","PhysicalEntity","CellularComponent","SarcoplasmicReticulum"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"f2f16":{"id":"f2f16","cellularComponent":"EndoplasmicReticulum","x":900,"y":2400,"color":"#ff0000","fontSize":10,"height":600,"rotation":0,"drawAs":"EndoplasmicReticulum","verticalAlign":"middle","width":600,"zIndex":32995,"textAlign":"center","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode","PhysicalEntity","CellularComponent","EndoplasmicReticulum"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"d8736":{"id":"d8736","cellularComponent":"GolgiApparatus","x":900,"y":3800,"color":"#ff0000","fontSize":10,"height":600,"rotation":0,"drawAs":"GolgiApparatus","verticalAlign":"middle","width":600,"zIndex":32997,"textAlign":"center","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode","PhysicalEntity","CellularComponent","GolgiApparatus"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"c6ccb":{"id":"c6ccb","x":900,"y":300,"color":"#ff0000","fontSize":10,"height":600,"rotation":0,"drawAs":"rect","borderRadius":15,"verticalAlign":"middle","width":600,"zIndex":32999,"textAlign":"center","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"f5713":{"id":"f5713","x":100.08470982653387,"y":1299.9694969774282,"fontSize":10,"height":300.6788711597367,"rotation":0,"drawAs":"rect","verticalAlign":"middle","width":599.8140755985673,"zIndex":33001,"textAlign":"center","color":"#000000","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"b505f":{"id":"b505f","x":100.68104819642895,"y":1740.1845522898154,"fontSize":10,"height":520.0372916892217,"rotation":0,"drawAs":"rect","verticalAlign":"middle","width":598.6046660892854,"zIndex":33002,"textAlign":"center","color":"#000000","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"c43a3":{"id":"c43a3","x":899.9999999999998,"y":1699.9249812453113,"fontSize":10,"height":599.5541854213554,"rotation":0,"drawAs":"rect","verticalAlign":"middle","width":600,"zIndex":33003,"textAlign":"center","color":"#000000","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"be35d":{"id":"be35d","x":198.4000000000001,"y":2408,"fontSize":10,"height":562.1333333333334,"rotation":0,"drawAs":"rect","verticalAlign":"middle","width":455.4666666666666,"zIndex":33004,"textAlign":"center","color":"#000000","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"db8d7":{"id":"db8d7","x":942.1768707482995,"y":2425.170068027211,"fontSize":10,"height":578.2312925170071,"rotation":0,"drawAs":"rect","verticalAlign":"middle","width":553.0612244897957,"zIndex":33005,"textAlign":"center","color":"#000000","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"b68ac":{"id":"b68ac","x":99.66777408637881,"y":3098.560354374308,"fontSize":10,"height":602.4363233665558,"rotation":0,"drawAs":"rect","verticalAlign":"middle","width":600.2214839424141,"zIndex":33006,"textAlign":"center","color":"#000000","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"e7b76":{"id":"e7b76","x":901.4396456256919,"y":3099.667774086379,"fontSize":10,"height":599.1140642303429,"rotation":0,"drawAs":"rect","verticalAlign":"middle","width":593.5769656699892,"zIndex":33007,"textAlign":"center","color":"#000000","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"ff5a5":{"id":"ff5a5","x":156.97674418604652,"y":3813.9534883720926,"fontSize":10,"height":572.0930232558147,"rotation":0,"drawAs":"rect","verticalAlign":"middle","width":543.0232558139535,"zIndex":33008,"textAlign":"center","color":"#000000","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"b32c8":{"id":"b32c8","x":961.6279069767442,"y":3832.558139534883,"fontSize":10,"height":539.5348837209306,"rotation":0,"drawAs":"rect","verticalAlign":"middle","width":503.48837209302366,"zIndex":33009,"textAlign":"center","color":"#000000","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"id3b9f67a":{"id":"id3b9f67a","color":"#cccccc","strokeDasharray":"5,3","borderWidth":1,"zIndex":-22,"points":[{"x":43.394489682048814,"y":4400},{"x":1521.1722674598261,"y":4400}],"drawAs":"StraightLine","type":["GraphicalLine","Edge"],"kaavioType":"Edge","gpmlElementName":"GraphicalLine"},"cbe54":{"id":"cbe54","color":"#cccccc","strokeDasharray":"5,3","borderWidth":1,"zIndex":-21,"points":[{"x":43.39448905987598,"y":3800},{"x":1521.1722668376533,"y":3800}],"drawAs":"StraightLine","type":["GraphicalLine","Edge"],"kaavioType":"Edge","gpmlElementName":"GraphicalLine"},"bd26b":{"id":"bd26b","color":"#cccccc","strokeDasharray":"5,3","borderWidth":1,"zIndex":-20,"points":[{"x":43.3945147731131,"y":300},{"x":1521.172292550891,"y":297.1794871794882}],"drawAs":"StraightLine","type":["GraphicalLine","Edge"],"kaavioType":"Edge","gpmlElementName":"GraphicalLine"},"fbba0":{"id":"fbba0","color":"#cccccc","strokeDasharray":"5,3","borderWidth":1,"zIndex":-19,"points":[{"x":43.39450423506054,"y":3700},{"x":1521.172282012838,"y":3700}],"drawAs":"StraightLine","type":["GraphicalLine","Edge"],"kaavioType":"Edge","gpmlElementName":"GraphicalLine"},"ea535":{"id":"ea535","color":"#cccccc","strokeDasharray":"5,3","borderWidth":1,"zIndex":-18,"points":[{"x":43.39449541284401,"y":3100},{"x":1521.1722731906216,"y":3100}],"drawAs":"StraightLine","type":["GraphicalLine","Edge"],"kaavioType":"Edge","gpmlElementName":"GraphicalLine"},"bb9f7":{"id":"bb9f7","color":"#cccccc","strokeDasharray":"5,3","borderWidth":1,"zIndex":-17,"points":[{"x":43.39448532683889,"y":3000},{"x":1521.1722631046164,"y":3000}],"drawAs":"StraightLine","type":["GraphicalLine","Edge"],"kaavioType":"Edge","gpmlElementName":"GraphicalLine"},"c3d22":{"id":"c3d22","color":"#cccccc","strokeDasharray":"5,3","borderWidth":1,"zIndex":-16,"points":[{"x":43.39453860038884,"y":2400},{"x":1521.1723163781662,"y":2400}],"drawAs":"StraightLine","type":["GraphicalLine","Edge"],"kaavioType":"Edge","gpmlElementName":"GraphicalLine"},"d0ad7":{"id":"d0ad7","color":"#cccccc","strokeDasharray":"5,3","borderWidth":1,"zIndex":-15,"points":[{"x":43.394511341441,"y":2300},{"x":1521.1722891192185,"y":2300}],"drawAs":"StraightLine","type":["GraphicalLine","Edge"],"kaavioType":"Edge","gpmlElementName":"GraphicalLine"},"d3372":{"id":"d3372","color":"#cccccc","strokeDasharray":"5,3","borderWidth":1,"zIndex":-14,"points":[{"x":43.39454416105866,"y":1700},{"x":1521.1723219388361,"y":1700}],"drawAs":"StraightLine","type":["GraphicalLine","Edge"],"kaavioType":"Edge","gpmlElementName":"GraphicalLine"},"c87b5":{"id":"c87b5","color":"#cccccc","strokeDasharray":"5,3","borderWidth":1,"zIndex":-13,"points":[{"x":43.394521879493624,"y":1600},{"x":1521.172299657271,"y":1600}],"drawAs":"StraightLine","type":["GraphicalLine","Edge"],"kaavioType":"Edge","gpmlElementName":"GraphicalLine"},"c70e2":{"id":"c70e2","color":"#cccccc","strokeDasharray":"5,3","borderWidth":1,"zIndex":-12,"points":[{"x":43.394485326838975,"y":1000},{"x":1521.1722631046175,"y":1000}],"drawAs":"StraightLine","type":["GraphicalLine","Edge"],"kaavioType":"Edge","gpmlElementName":"GraphicalLine"},"eca55":{"id":"eca55","color":"#cccccc","strokeDasharray":"5,3","borderWidth":1,"zIndex":-11,"points":[{"x":43.39454759273079,"y":900},{"x":1521.1723253705084,"y":900}],"drawAs":"StraightLine","type":["GraphicalLine","Edge"],"kaavioType":"Edge","gpmlElementName":"GraphicalLine"},"id41348e99":{"id":"id41348e99","color":"#cccccc","strokeDasharray":"5,3","borderWidth":1,"zIndex":-9,"points":[{"x":700,"y":185.55559355115136},{"x":700,"y":4774.444482440042}],"drawAs":"StraightLine","type":["GraphicalLine","Edge"],"kaavioType":"Edge","gpmlElementName":"GraphicalLine"},"fa68f":{"id":"fa68f","color":"#cccccc","strokeDasharray":"5,3","borderWidth":1,"zIndex":-8,"points":[{"x":100,"y":185.556},{"x":100,"y":4774.4444980318585}],"drawAs":"StraightLine","type":["GraphicalLine","Edge"],"kaavioType":"Edge","gpmlElementName":"GraphicalLine"},"ca2cc":{"id":"ca2cc","color":"#cccccc","strokeDasharray":"5,3","borderWidth":1,"zIndex":-7,"points":[{"x":900,"y":185.5556247347877},{"x":905.4700854700845,"y":4774.444513623677}],"drawAs":"StraightLine","type":["GraphicalLine","Edge"],"kaavioType":"Edge","gpmlElementName":"GraphicalLine"},"ea33b":{"id":"ea33b","color":"#cccccc","strokeDasharray":"5,3","borderWidth":1,"zIndex":-6,"points":[{"x":1500,"y":185.55555555555554},{"x":1501.794871794869,"y":4774.444444444444}],"drawAs":"StraightLine","type":["GraphicalLine","Edge"],"kaavioType":"Edge","gpmlElementName":"GraphicalLine"}}}
